resources/qml/dialogs/RoomSettings.qml

```diff
@@ -254,6 +254,22 @@ ApplicationWindow {
                 Layout.alignment: Qt.AlignRight
             }
 
+            MatrixText {
+                text: qsTr("Hidden events")
+            }
+
+            HiddenEventsDialog {
+                id: hiddenEventsDialog
+                prompt: qsTr("Select the events you want to hide from %1").arg(roomSettings.roomName)
```

Comment:
```suggestion
                prompt: qsTr("Select the events you want to hide in %1").arg(roomSettings.roomName)
```

The screenshot was very confusing before I saw this :D 

(Since I thought you wanted to hide it from the client)

Reply:
Yeah, you're right. Setting something to *on* to turn it *off* is confusing. 😄